templates/firefox-ios.xliff

```diff
@@ -1355,9 +1355,38 @@
         <source>Logins</source>
         <note>Toggle logins syncing setting</note>
       </trans-unit>
+      <trans-unit id="Logins.DevicePasscodeRequired.LearnMoreButtonTitle" xml:space="preserve">
+        <source>Learn More</source>
+        <note>Title of the Learn More button that links to a support page about device passcode requirements.</note>
+      </trans-unit>
+      <trans-unit id="Logins.DevicePasscodeRequired.Message" xml:space="preserve">
+        <source>To save and autofill logins and passwords, enable Face ID, Touch ID or a device passcode.</source>
+        <note>Message shown when you enter Logins &amp; Passwords without having a device passcode set.</note>
+      </trans-unit>
+      <trans-unit id="Logins.Onboarding.ContinueButtonTitle" xml:space="preserve">
+        <source>Continue</source>
+        <note>Title of the Continue button.</note>
+      </trans-unit>
+      <trans-unit id="Logins.Onboarding.LearnMoreButtonTitle" xml:space="preserve">
+        <source>Learn More</source>
+        <note>Title of the Learn More button that links to a support page about device passcode requirements.</note>
+      </trans-unit>
+      <trans-unit id="Logins.Onboarding.Message" xml:space="preserve">
+        <source>Your logins and passwords are now protected by Face ID, Touch ID or a device passcode.</source>
+        <note>Message shown when you enter Logins &amp; Passwords for the first time.</note>
+      </trans-unit>
+      <trans-unit id="Logins.PasscodeRequirement.Warning" xml:space="preserve">
+        <source>To use the AutoFill feature for Firefox, you must have a device passcode enabled.</source>
+        <note>Warning message shown when you try to enable or use native AutoFill without a device passcode setup</note>
+      </trans-unit>
       <trans-unit id="Logins.WelcomeView.Title" xml:space="preserve">
```

Comment:
String ID should be updated since string changes so much.
Also, seems like there are two comments below, instead of one

Reply:
Fixed with https://github.com/mozilla-mobile/firefox-ios/pull/9488